2024-RIKEN-AWS/JupyterNotebook/docker/Dockerfile.spawn

```diff
@@ -79,11 +101,17 @@ RUN python3 -m pip install jupyter_app_launcher && \
 COPY ./docker/jupyter-launcher.yaml /usr/local/share/jupyter/lab/jupyter_app_launcher/config.yaml
 ENV JUPYTER_APP_LAUNCHER_PATH /usr/local/share/jupyter/lab/jupyter_app_launcher
 
-# No permission errors here
-USER ${NB_USER}
+# Give jovyan user permissions to tutorial materials
+RUN chmod -R 777 ~/flux-tutorial-2024
+
 WORKDIR $HOME
 COPY ./docker/flux-icon.png $HOME/flux-icon.png
 
+# RUN ${VENV_SOURCE} && \
```

Comment:
What is this commented out section?

Reply:
Oh, I should remove that. It was stuff we had from trying to get DLIO to work.